identity-service/src/audiusLibsInstance.js

```diff
@@ -30,7 +31,8 @@ class AudiusLibsWrapper {
           ownerWallet: config.get('relayerPublicKey')
         }
       },
-      isServer: true
+      isServer: true,
+      captchaConfig: { serviceKey: '6LfVR-0ZAAAAAGN7YqtPC7DyVzgsLc_4_9_5MyVj' }
```

Comment:
maybe throw this in a secret for now too

Reply:
doing this still?

Reply:
yes! just added & will redeploy tomorrow for testing